DQM/DTMonitorClient/src/DTNoiseAnalysisTest.cc

```diff
@@ -75,11 +76,57 @@ void DTNoiseAnalysisTest::beginRun(Run const& run, EventSetup const& context) {
 
   if (!bookingdone) {
     // book the histos
-  bookHistos(ibooker);
+    bookHistos(ibooker);
+
+   //How many channels per sector?
+
+    for (int isec=0; isec<12; isec++) {
+      NwSec[isec]=0;
+
+      for (int ist=0; ist<4; ist++) {
+        NwCh[isec][ist]=0;
+
+        DTChamberId chID (0, ist+1, isec+1);
+        for (int isl=1; isl<4; isl++) {
+
+         if ( chID.station() == 4 && isl ==2 ) continue;
+         const DTSuperLayer * SL = (muonGeom->chamber(chID))->superLayer(isl);
+         for (int ilay=1; ilay<5; ilay++) { 
+          int nw =  ((SL->layer(ilay))->specificTopology()).channels();
+          NwCh[isec][ist]+=nw;
+          NwSec[isec]+=nw;
+         }
+        } // on superlayers
+      } // on stations
+    } // on sectors
+
+    // add Sec 13 and 14
+    DTChamberId chID (0, 4, 13);
+    for (int isl=1; isl<4; isl++) {
+
+      if (isl ==2) continue;
+      const DTSuperLayer * SL = (muonGeom->chamber(chID))->superLayer(isl);
+      for (int ilay=1; ilay<5; ilay++) { 
+       int nw =  ((SL->layer(ilay))->specificTopology()).channels();
+       NwCh[3][3]+=nw;
```

Comment:
hi, sorry I took so long!
Yes: it is wanted. And also: NwCh[9][3]+=nw;  at line 123!
[3][3] means: sector 4, station 4. This DT station (and also MB4/Sect10, i.e. [9][3] here) is actually a pair of stations and in the software the twins are often treated as one, as it is the case in this class. You see for instance:
 if(sector == 13) {
                sector = 4;
              } else if(sector == 14) {
                sector = 10;
              }
https://github.com/cms-sw/cmssw/blob/CMSSW_8_1_X/DQM/DTMonitorClient/src/DTNoiseAnalysisTest.cc#L140

13 and 14 are the fictitious sector numbers of the twins. So here I am adding the wires of MB4/Sect13 to MB4/Sect4 and MB4/Sect14 to MB4/Sec10, to properly handle them as one, as they are in the rest of the code.